src/ro/siit/LocalStore/Store.java

```diff
@@ -0,0 +1,158 @@
+package ro.siit.LocalStore;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+
+import static ro.siit.LocalStore.Main.productList;
+import static ro.siit.LocalStore.SoldItem.soldList;
+import static ro.siit.LocalStore.StockItem.stockList;
+
+public class Store {
+
+    //Adding a product
+    static int id = 4;
+    public static void addProduct() {
+        id++;
+        System.out.println("What type is the product that you want to add? Type 1 for Animal, 2 for Vegetable");
+        Scanner sc = new Scanner(System.in);
+        String choice = sc.next();
+
+        //Verifying if the input is 1 (for animal) or 2 (for vegetable)
+        if (!(choice.equals("1") || choice.equals("2"))){
+            System.out.println("This is not an option");
+            System.exit(0);
+        }
+
+        System.out.println("Type the name of the product");
+        String verif = sc.next();
+
+        //Verifying if the input is one of the animal product that can be added in store
+        switch (choice){
+            case "1":
+                if (verif.equals("cheese") || verif.equals("egg") || verif.equals("milk") || verif.equals("yogurt") || verif.equals("sour cream")){
+                    String name = verif;
+                }
+                else {
+                    System.out.println("This is not an animal product");
+                    System.exit(0);
+                }
+                break;
+            case "2":
+                String name = verif;
+                break;
+        }
+
+        System.out.println("Type the price of the product");
+        Double price = Double.parseDouble(sc.next());
+
+        System.out.println("Type the weight of the product");
+        int weight = Integer.parseInt(sc.next());
+
+        System.out.println("Type the quantity of the product");
+        int quantity = Integer.parseInt(sc.next());
+
+        switch (choice){
+            case "1":
+                System.out.println("Type the storage temperature");
+                int storageTemp = Integer.parseInt(sc.next());
+
+                //Defining the product and adding it in productList
+                Product p5 = new Animal(id, verif , price, weight, storageTemp, quantity);
+                productList.add(p5);
+                break;
+            case "2":
+                System.out.println ("Type the list of vitamins the product contains");
+                String vitamin =sc.next();
+
+                //Defining the product and adding it in productList
+                Product p6 = new Vegetable(id, verif , price, weight, vitamin, quantity);
+                productList.add(p6);
+                break;
+        }
+    }
+
+    //Adding the products in stockList
+
+    public static void addStock() {
```

Comment:
why do this every time before selling? why not just do it when adding the product only for that specific product?